code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -1,4 +1,4 @@
-var/list/hardcoded_gases = list("o2","n2","co2","plasma") //the main four gases, which were at one time hardcoded
+var/list/hardcoded_gases = list("o2","n2","co2","plasma","water vapor","freon","n2o") 
```

Comment:
N2 is nitrogen, n2o is nitrous oxide. They are different gases and you can confirm that by scanning any n2o canister outside of this PR and seeing that it comes back with no list of contents. Whatever ill make a snowflake list for atmos scanner.